keyboards/bt66tech/bluepill/ws2812.c

```diff
@@ -0,0 +1,133 @@
+/*
```

Comment:
This should be removed, and migrated to use <https://docs.qmk.fm/#/ws2812_driver?id=spi>